src/adagrams.js

```diff
@@ -1,8 +1,97 @@
 const Adagrams = {
   drawLetters() {
     // Implement this method for wave 1
+    const letterPool = ( "A" * 9 + "B" * 2 + "C" * 2 + "D" * 4 + "E" * 12 + "F" * 2 + "G" * 3 +
+     "H" * 2 + "I" * 9 + "J" + "K" + "L" * 4 + "M" * 2 + "N" * 6 + "O" * 8 + "P" * 2 +
+      "Q" + "R" * 6 + "S" * 4 + "T" * 6 + "U" * 4 + "V" * 2 + "W" * 2 + "X" +
+       "Y" * 2 + "Z").toUpperCase().split('');
+
+    shuffle(letterPool)
```

Comment:
Here, you shuffle the entire array pool of letters... and then, you take the first ten. Consider how you could refactor this so that you randomly pick ten letters, instead of shuffling the entire array.